src/pkg/filters/filters_test.go

```diff
@@ -206,6 +206,31 @@ func (suite *FiltersSuite) TestPrefixes() {
 	}
 }
 
+func (suite *FiltersSuite) TestSuffixes() {
+	target := "folderB"
+	f := filters.Suffix(target)
+	nf := filters.NotSuffix(target)
+
+	table := []struct {
+		name     string
+		input    string
+		expectF  assert.BoolAssertionFunc
+		expectNF assert.BoolAssertionFunc
+	}{
+		{"Exact match - same case", "folderB", assert.True, assert.False},
+		{"Exact match - different case", "Folderb", assert.True, assert.False},
+		{"Suffix match - same case", "folderA/folderB", assert.True, assert.False},
+		{"Suffix match - different case", "Foldera/folderb", assert.True, assert.False},
```

Comment:
should probably have a case to check it matches `foldera/afolderb`

Reply:
The case difference is in the suffix.  "folder`B`" - "folder`b`"